libs/coin-modules/coin-tron/src/bridge/getEstimateFees.ts

```diff
@@ -24,23 +24,48 @@ const getFeesFromBandwidth = (account: Account, transaction: Transaction): BigNu
 };
 
 // Special case: If activated an account, cost around 0.1 TRX
-const getFeesFromAccountActivation = async (a: Account, t: Transaction): Promise<BigNumber> => {
-  const recipientAccount = await fetchTronAccount(t.recipient);
+const getFeesFromAccountActivation = async (
+  a: Account,
+  t: Transaction,
+  ta?: TokenAccount | null,
```

Comment:
Use more understandable variable names:

```suggestion
  account: Account,
  transaction: Transaction,
  tokenAccount?: TokenAccount | null,
```